grants_tagger/bertmesh/model.py

```diff
@@ -55,4 +56,7 @@ def forward(self, input_ids, **kwargs):
             outs = torch.nn.functional.relu(self.linear_1(cls))
             outs = self.dropout_layer(outs)
             outs = torch.sigmoid(self.linear_out(outs))
+        if return_labels:
+            # TODO Vectorize
+           outs =  [[self.id2label[label_id] for label_id, label_prob in enumerate(out) if label_prob > 0.5] for out in outs]
```

Comment:
I think this might not work for batch size > 1 ?

Reply:
Hmm, I can test this but why would it not work? `for out in outs` should handle that